packages/@studiocms/mdx/vitest.config.ts

```diff
@@ -0,0 +1,35 @@
+import type { AstroIntegration } from 'astro';
+import { getViteConfig } from 'astro/config';
+import { addVirtualImports } from 'astro-integration-kit';
+import { defineProject } from 'vitest/config';
+import { studiocmsMDX } from './src/index.js';
+
+const testIntegration: AstroIntegration = {
+	name: 'test-integration',
+	hooks: {
+		'astro:config:setup': (params) => {
+			addVirtualImports(params, {
+				name: 'test-integration',
+				imports: {
+					'studiocms:component-registry/runtime':
+						// Test-only identity renderer: mirrors API shape but skips sanitization on purpose.
+						'export const createRenderer = (result, sanitize, preRenderer) => (content) => content;',
+				},
+			});
+		},
+	},
+};
+
+export default defineProject(
+	getViteConfig(
+		{
+			test: {
+				environment: 'node',
+				include: ['**/*.test.ts'],
+			},
+		},
+		{
+			integrations: [testIntegration, studiocmsMDX()],
```

Comment:
```suggestion
			integrations: [testIntegration],
```

You can't use a StudioCMS Plugin as an Astro integration, You'll need to extract the integration like how the tests for MD/HTML do.